services/static-webserver/client/source/class/osparc/widget/IntlTelInput.js

```diff
@@ -118,8 +118,14 @@ qx.Class.define("osparc.widget.IntlTelInput", {
 
     __inputToPhoneInput: function(input) {
       const iti = intlTelInput(input, {
-        initialCountry: "ch", // auto: geoIpLookup. need to unlock https://ipinfo.io/,
-        preferredCountries: ["ch", "us"]
+        initialCountry: "auto",
+        geoIpLookup: callback => {
+          fetch("https://ipapi.co/json")
+            .then(res => res.json())
```

Comment:
can you please also pass the response of this call  when you submit the form as a field e.g. ` "ipapi" : {...}`

Reply:
I could pass it, but I read this endpoint has some limitations. The frontend uses it for making the UX a bit better, while the backend really needs the information, I wouldn't rely on the frontend's request 👍.